tests/conftest.py

```diff
@@ -3,7 +3,9 @@
 import pytest
 import sqlalchemy
 
-TEST_SETTINGS = {"database_url": os.environ.get("TEST_DATABASE_URL", "sqlite://")}
+TEST_SETTINGS = {
+    "database_url": os.environ["TEST_DATABASE_URL"],
+}
```

Comment:
`conftest.py` can assume that `TEST_DATABASE_URL` is set correctly because `tox.ini` ensures it